GANDLF/entrypoints/run.py

```diff
@@ -0,0 +1,291 @@
+    raw_input: str,
+):
+    """Semantic segmentation, regression, and classification for medical images using Deep Learning."""
+    _run(
+        config=config,
+        input_data=input_data,
+        train_flag=train,
+        model_dir=model_dir,
+        device=device,
+        reset_flag=reset,
+        resume_flag=resume,
+        output_path=output_path,
+    )
+
+
+@deprecated(
+    "This is a deprecated way of running GanDLF. Please, use `gandlf run` cli command "
+    + "instead of `gandlf_run`. Note that in new CLI tool some params were renamed or changed its behavior:\n"
+    + "  --parameters_file to --config\n"
+    + "  --inputdata/--data_path to --input-data\n"
+    + "  --train changed its behavior: instead of `--train True/False` pass `--train/--infer`\n"
+    + "  --modeldir to --model-dir\n"
+    + "  --reset is flag now with default False value if not passed\n"
+    + "  --resume is flag now with default False value if not passed\n"
+    + "  --outputdir/--output_path to --output-path; in training mode, use --model-dir instead\n"
+    + "  --version removed; use `gandlf --version` instead\n"
+    + "`gandlf_run` script would be deprecated soon."
+)
+def old_way():
+    logger_setup()
+    parser = argparse.ArgumentParser(
+        prog="GANDLF",
+        formatter_class=argparse.RawTextHelpFormatter,
+        description="Semantic segmentation, regression, and classification for medical images using Deep Learning.\n\n"
+        + copyrightMessage,
+    )
+    parser.add_argument(
+        "-c",
+        "--config",
+        "--parameters_file",
+        metavar="",
+        type=str,
+        required=True,
+        help="The configuration file (contains all the information related to the training/inference session)",
+    )
+    parser.add_argument(
+        "-i",
+        "--inputdata",
+        "--data_path",
+        metavar="",
+        type=str,
+        required=True,
+        help="Data CSV file that is used for training/inference; can also take comma-separated training-validation pre-split CSVs",
+    )
+    parser.add_argument(
+        "-t",
+        "--train",
+        metavar="",
+        type=ast.literal_eval,
+        required=True,
+        help="True: training and False: inference; for inference, there needs to be a compatible model saved in '-modeldir'",
+    )
+    parser.add_argument(
+        "-m",
+        "--modeldir",
+        metavar="",
+        type=str,
+        help="Training: Output directory to save intermediate files and model weights; inference: location of previous training session output",
+    )
+    parser.add_argument(
+        "-d",
+        "--device",
+        default="cuda",  # TODO: default value doesn't work as arg is required
+        metavar="",
+        type=str,
+        required=True,
+        help="Device to perform requested session on 'cpu' or 'cuda'; for cuda, ensure CUDA_VISIBLE_DEVICES env var is set",
+    )
+    parser.add_argument(
+        "-rt",
```

Comment:
maybe aslo reset and resume flags could use `store_true` flag to avoid confusing string evaluation? Although this is being deprecated, perhaps some old users may have some scripts done and are aware of that - open for discussion if this should be modiifed

Reply:
I am partial to not making major changes to the argeparse interfaces (since we will be deprecating them at a future release anyway)... Thoughts?

Reply:
Yeah, it would be a breaking change. If so, comment above is also irrelevant. Closing